dmcontent/content_loader.py

```diff
@@ -249,6 +261,41 @@ def get_previous_question_id(self, question_id):
 
         return None
 
+    def get_next_question_slug(self, question_slug=None):
```

Comment:
Should we call these `get_next_question` and make them return the question object instead of the `.slug` attribute?

I also wonder if we can do anything to merge them with the `get_next_question_id` methods. It feels like the only difference between them is the attribute name we use for lookup.